pipeline/ingest/behavior.py

```diff
@@ -466,6 +466,11 @@ def make(self, key):
                 outcome = 'ignore'
 
             bkey['outcome'] = outcome
+
+            # Determine free/autowater
+            bkey['auto_water'] = True if gui['Autowater'][0] == 2 else False
```

Comment:
Shouldn't this be: 
`bkey['auto_water'] = True if gui['Autowater'][0] == 1 else False`

Reply:
per discussion with susu, 2 is enabled and 1 is disabled- not sure why these values were chosen.

Reply:
will make a comment.

Reply:
I think it's the other way around:
``` When SessionData.TrialSettings.GUI.Autowater = 1, autowater on, when SessionData.TrialSettings.GUI.Autowater = 2, autowater off.```